filename: backsdk4.css
checksum: 76aebb38a99da6181ba26735ef82b368f14866550d267fb75fce00da57054fab

--- backsdk4.css	2022-09-21 23:53:42.000000000 +0100
+++ backsdk4.css	2023-10-08 11:57:43.154276043 +0100
@@ -104,7 +104,7 @@
 ul ol, ol ol
 	{
 	list-style-type: lower-alpha;
-	{
+	}
 
 pre
 	{
